IsCourseActivitySyncEnabled for LearningProvider (#21464)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/employeeexperience-post-learningproviders.md

|Property|Type|Description|
|:---|:---|:---|
|displayName|String|The display name that appears in Viva Learning. Required.|

> |isCourseActivitySyncEnabled|Boolean|Indicates whether a provider can ingest [learningCourseActivity](../resources/learningcourseactivity.md) records. The default value is `false`. Set to true to make learningCourseActivities available for this provider.|

Reviewer: Please format the word true as a value: `true`

Reviewer 2: Updated by author. Thanks.

|longLogoWebUrlForLightTheme|String|The long logo URL for the light mode that needs to be a publicly accessible image. This image would be saved to the blob storage of Viva Learning for rendering  within the Viva Learning app. Required.|
|squareLogoWebUrlForDarkTheme|String|The square logo URL for the dark mode that needs to be a publicly accessible image. This image would be saved to the blob storage of Viva Learning for rendering within the Viva Learning app. Required.|
|squareLogoWebUrlForLightTheme|String|The square logo URL for the light mode that needs to be a publicly accessible image. This image would be saved to the blob storage of Viva Learning for rendering within the Viva Learning app. Required.|

> |isCourseActivitySyncEnabled|Boolean|Indicates whether a provider can ingest [learningCourseActivity](../resources/learningcourseactivity.md) records. The default value is `false`. Set to true to make learningCourseActivities available for this provider.|

Reviewer: Global: Please add the new property to the table in alphabetical order instead of adding it to the end of the table.

> |isCourseActivitySyncEnabled|Boolean|Indicates whether [learningCourseActivity](../resources/learningcourseactivity.md) records will be ingested for this provider or not. Default value is `false`. Learning course activities for a provider cannot be ingested if this value is false. This value needs to be updated to true when the provider wants to ingest learningCourseActivities.|

Reviewer 2: ```suggestion
|isCourseActivitySyncEnabled|Boolean|Indicates whether a provider can ingest [learningCourseActivity](../resources/learningcourseactivity.md) records. The default value is `false`. Set to true to make learningCourseActivities available for this provider.|
```
